rules/windows/credential_access_iis_connectionstrings_dumping.toml

```
@@ -0,0 +1,48 @@
+[metadata]
+creation_date = "2020/08/18"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/08/18"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies use of aspnet_regiis to decrypt Microsoft IIS connection strings. An attacker with Microsoft IIS web server
+access via a webshell or alike can decrypt and dump any hardcoded connection strings, such as the MSSQL service account
+password using aspnet_regiis command.
+"""
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "kuery"
+license = "Elastic License"
+max_signals = 33
+name = "Microsoft IIS Connection Strings Decryption"
+references = [
+    "https://blog.netspi.com/decrypting-iis-passwords-to-break-out-of-the-dmz-part-1/",
+    "https://symantec-enterprise-blogs.security.com/blogs/threat-intelligence/greenbug-espionage-telco-south-asia",
+]
+risk_score = 73
+rule_id = "c25e9c87-95e1-4368-bfab-9fd34cf867ec"
+severity = "high"
+tags = ["Elastic", "Windows"]
+type = "query"
+
+query = '''
+event.category:process and event.type:(start or process_started) and
+  (process.name:aspnet_regiis.exe or process.pe.original_file_name:aspnet_regiis.exe) and 
+  process.args:(connectionStrings and "-pdf")
+'''
+
+
+[[rule.threat]]
+framework = "MITRE ATT&CK"
+[[rule.threat.technique]]
+id = "T1003"
+name = "Credential Dumping"
```

```suggestion
name = "OS Credential Dumping"
```

ATT&CK was refreshed and so the name needs updating